api/src/main/java/ca/bc/gov/educ/eas/api/service/v1/AssessmentStudentService.java

```diff
@@ -50,7 +51,8 @@ public AssessmentStudentEntity updateStudent(AssessmentStudentEntity assessmentS
 
     @Transactional(propagation = Propagation.REQUIRES_NEW)
     public AssessmentStudentEntity createStudent(AssessmentStudentEntity assessmentStudentEntity) {
-        return createAssessmentStudentWithHistory(assessmentStudentEntity);
+        Optional<AssessmentStudentEntity> studentEntity = assessmentStudentRepository.findByAssessmentEntity_AssessmentIDAndStudentID(assessmentStudentEntity.getAssessmentEntity().getAssessmentID(), assessmentStudentEntity.getStudentID());
```

Comment:
We'll need to push this logic into the SAGA function. We don't want the same thing occurring if someone else uses the function in the service.